lib/mongify/database/sql_connection.rb

```diff
@@ -100,7 +102,19 @@ def select_paged_rows(table_name, batch_size, page)
                         WHERE rnum > #{offset}"
           )
         end
-        connection.select_all("SELECT * FROM #{table_name} LIMIT #{batch_size} OFFSET #{(page - 1) * batch_size}")
+
+        q_select = "SELECT t.* FROM #{table_name} t"
+
+        # TODO: How to handle orderby for tables that dont have an id to order by.
+        q_batch = "ORDER BY id LIMIT #{batch_size} OFFSET #{(page - 1) * batch_size}"
+        q_sync = "#{SYNC_HELPER_TABLE} u WHERE t.updated_at > u.last_updated_at AND u.table_name = '#{table_name} '"
+
+        q = q_select + ', ' + q_batch
+        if is_sync
+            q = q_select + ', ' + q_sync + ' ' + q_batch
```

Comment:
Use 2 (not 4) spaces for indentation.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.